### PR TITLE
Feature: Add `home_total_amount` to the `stg_quickbooks__deposit` model

### PR DESCRIPTION
**Pull request by fivetran-avinash**
## PR Overview
**This PR will address the following Issue/Feature:** Supporting issues filed in `dbt_quickbooks`, particularly [115](https://github.com/fivetran/dbt_quickbooks/issues/115) and [128](https://github.com/fivetran/dbt_quickbooks/issues/128) 

**This PR will result in the following new package version:**  v0.10.1

We are adding a new field, but it should not impact existing customers to be breaking. (Given the scale of the concurrent PR in the transform package, I'd be open to making this breaking though)

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

## 🎉 Feature Update 🎉
- Added the `home_total_amount` field into the  `stg_quickbooks__deposit` model to support the new multicurrency feature in the [v0.14.0 release of the `dbt_quickbooks` package](https://github.com/fivetran/dbt_quickbooks/releases/tag/v0.14.0).

## 🚘 Under the Hood 🚘
- Included auto-releaser GitHub Actions workflow to automate future releases.
- Updated the maintainer PR template to resemble the most up to date format.
- Removed the check docs GitHub Action as it is no longer necessary.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [NA] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [NA] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
    
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

Validation details are in the transformation package, but did confirm the new column is being applied correctly in the `dbt_quickbooks` [PR #134](https://github.com/fivetran/dbt_quickbooks/pull/134).

### If you had to summarize this PR in an emoji, which would it be?
 
💴 💷 💶

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# dbt_quickbooks_source v0.10.1
+[PR #53](https://github.com/fivetran/dbt_quickbooks_source/pull/53) introduces the following updates: 
+
+## 🎉 Feature Update 🎉
+- Added the `home_total_amount` field into the  `stg_quickbooks__deposit` model to support the new multicurrency feature in the [v0.14.0 release of the `dbt_quickbooks` package](https://github.com/fivetran/dbt_quickbooks/releases/tag/v0.14.0).
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:5]:
```suggestion
- Added the `home_total_amount` field into the `stg_quickbooks__deposit` model to support the new multicurrency feature in the [v0.14.0 release of the `dbt_quickbooks` package](https://github.com/fivetran/dbt_quickbooks/releases/tag/v0.14.0).
```

File: models/stg_quickbooks__deposit.sql [hunk 2]
```diff
@@ -44,6 +44,7 @@ final as (
         currency_id,
         cast(department_id as {{ dbt.type_string() }}) as department_id,
         total_amount,
+        home_total_amount,
```
Comment (fivetran-joemarkiewicz) [inline models/stg_quickbooks__deposit.sql:47]: Please be sure to add this to the [src](https://github.com/fivetran/dbt_quickbooks_source/blob/f9fb58596c28c395a4d45ac1e42fb0f3e2d694b6/models/src_quickbooks.yml#L363-L384) and [stg.yml](https://github.com/fivetran/dbt_quickbooks_source/blob/f9fb58596c28c395a4d45ac1e42fb0f3e2d694b6/models/stg_quickbooks.yml#L458-L487) files and regen the docs.

File: models/stg_quickbooks__estimate.sql [hunk 3]
```diff
@@ -45,6 +45,7 @@ final as (
         customer_id,
         cast(department_id as {{ dbt.type_string() }}) as department_id,
         cast( {{ dbt.date_trunc('day', 'due_date') }} as date) as due_date, 
+        exchange_rate,
```
Comment (fivetran-joemarkiewicz) [inline models/stg_quickbooks__estimate.sql:48]: Please be sure to add this to the [src](https://github.com/fivetran/dbt_quickbooks_source/blob/f9fb58596c28c395a4d45ac1e42fb0f3e2d694b6/models/src_quickbooks.yml#L386-L411) and [stg.yml](https://github.com/fivetran/dbt_quickbooks_source/blob/f9fb58596c28c395a4d45ac1e42fb0f3e2d694b6/models/stg_quickbooks.yml#L489-L522) files and regen the docs.

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
This update looks good to me, I just have some documentation requests to be applied before approving. Let me know once this is ready for re-review. Thanks!

**Review by fivetran-avinash — COMMENTED** [review]
@fivetran-joemarkiewicz Changes applied and docs regenerated, ready for re-review.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM

**Review by fivetran-catfritz — APPROVED** [review]
LGTM!

**Pull request merged**